aksel.nav.no/website/pages/eksempler/primitive-page/demo.tsx

```diff
@@ -1,32 +1,28 @@
-import { Box, Page } from "@navikt/ds-react";
+import { Heading, Page } from "@navikt/ds-react";
+import { Content } from "@/web/examples/__parts/PageDemoContent";
```

Comment:
Alias in imports from __parts doesn't work
```suggestion
import { Content } from "../../../components/website-modules/examples/__parts/PageDemoContent";
```